cmd/entrypoint/namespaces_test.go

```diff
@@ -0,0 +1,38 @@
+// +build linux
+
+package main
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+// This isn't a great unit test, but it's the best I can think of.
+// It can only run on Linux, and tries to make a network connection.
+func TestDropNetworking(t *testing.T) {
```

Comment:
What's the expected behaviour wrt Sidecars alongside Steps?  These share a `localhost` under normal circumstances in a Pod I think?

My assumption reading this is that `localhost` of one process is not accessible to another process in new networking namespace, is that right?  Worth unit testing that behaviour as well with a local http server running in a separate process?

Reply:
Good question, I think for that we'll need an actual e2e test to really simulate the behavior of the containers in a Pod. I'll make sure we include that case.